journal/202106.md

```diff
@@ -0,0 +1,464 @@
+
+{ intro, major news of the month, don't put any links here
+
+- new consensus upgrade on the horizon
+- approved proposals
+- treasury consensus bug discovered
+- hashrate drop
+
+}
+
+Contents: { sync }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+
+## Treasury bug recap
+
+{ what is not affected, what is affected, what to expect }
+
+- https://blog.decred.org/2021/06/25/Treasury-Expenditure-Policy-Bug/
+- https://twitter.com/matheusd_tech/status/1409928455974699013
+- https://twitter.com/lukebp_/status/1409929016400822279
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+The way standard scripts are handled has been reworked to address a number of long-standing issues.
+
+For a quick background, "[scripts](https://devdocs.decred.org/developer-guides/transactions/txscript/overview/)" are small commands stored inside [transactions](https://devdocs.decred.org/developer-guides/transactions/transaction-format/) that consume "input" coins from past transactions and create new "output" coins for the recipient. "Standard scripts" is a subset of all scripts allowed by the consensus that covers most useful operations (sending funds, staking, multisig, etc) optimized for efficiency and network security. Mainnet nodes will normally [refuse](https://github.com/decred/dcrd/pull/2656#issuecomment-851105968) to accept and relay standalone transactions (those that are not part of a block) containing non-standard scripts.
+
+The [reworked code](https://github.com/decred/dcrd/pull/2656) adds a new package called `stdscript` that supports different script versions (this was hard to support previously and is required for the next consensus upgrade), provides a more ergonomic API for developers (including multisig and atomic swap helpers) and improves separation between "standard" and "consensus" (a common point of confusion for new developers). Additionally, the policy of what scripts are considered "standard" has been further tightened to help ensure the typical scripts occupy less space on-chain and further restrict cases that don't make sense.
+
+As usual, this large change comes in a series of easy-to-digest commits with informative descriptions, tests, benchmarks, examples, all finished with a thorough new [README](https://github.com/decred/dcrd/blob/master/internal/staging/stdscript/README.md).
```

Comment:
I suggest using a pinned commit hash for this link because it's in the staging area which means it's going to move and without a pinned hash this link will break:
```suggestion
As usual, this large change comes in a series of easy-to-digest commits with informative descriptions, tests, benchmarks, examples, all finished with a thorough new [README](https://github.com/davecgh/dcrd/blob/62950c2b8b8d831cacf6f2ec46216e420d2658cc/internal/staging/stdscript/README.md).
```